### ISSUE TITLE
Descriptive labels missing on radio buttons (WCAG 2.2 SC 3.3.2 Labels or Instructions)

### ISSUE THREAD

**Issue by Aurelia-LSE** [amandadoughty/moodle-mod_peerwork#36]
The radio buttons in the "Grade your peers" do not have a meaningful label associated to them. Screen reader users (tested with NVDA) don't know what option they are selecting when navigating them. This fails WCAG 2.2 SC 3.3.2 Labels or Instructions.

![Image](https://github.com/user-attachments/assets/59e43183-efc0-4396-b200-4b553aa23f0d)

Perhaps, instead of adding a `<span class="sr-only">Rating &#039;Not yet competent&#039; for Test Student</span>`, the accessible name could be given with [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-label) ? 

![Image](https://github.com/user-attachments/assets/0c28a925-68cc-4ec1-89fe-887c28c5ec38)
